src/core/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = {
+  'rules': {
+    'no-restricted-imports': [
+      'error',
+      {
+        // Disallow importing dependencies from outside this directory
+        'patterns': ['..'],
```

Comment:
I'm not certain that this works correctly. Can you verify with several imports?

Reply:
Indeed I did!

![image](https://user-images.githubusercontent.com/6694512/109209108-262a8100-7779-11eb-84cf-d24981619fbe.png)
![image](https://user-images.githubusercontent.com/6694512/109209139-2fb3e900-7779-11eb-8000-4b6f22108323.png)